src/Commands.pm

```diff
@@ -75,6 +75,7 @@ sub initHandlers {
 	buyer				=> \&cmdBuyer,
 	bs					=> \&cmdBuyShopInfoSelf,
 	c					=> \&cmdChat,
+	cancel				=> \&cmdCancel,
```

Comment:
`cancel` is a very generic term, which could mean a lot of things which aren't related to buying/selling. I recommend  `buy cancel` and `sell cancel`, even if they do the same thing, and even though `store` and `sell` are top-level commands. Though only `sell` matches its sub-command name, so `store` is pretty messed up too - it probably should be named `buy list` or something.